luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/FactoryPark.java

```diff
@@ -14,14 +14,14 @@
  */
 public class FactoryPark<T> {
 
-    static final String ENTITY_TYPE = "factory park";
+    private static final String ENTITY_TYPE = "factory park";
 
     private static final String FACTORY_KEY = "type";
 
     private static final String UNKNOWN_FACTORY_NAME = "factory name '%s' in config is not known to the Luthier module";
 
     // Use a supplier to support deferred loading
-    protected final Supplier<ObjectNode> configSource;
+    private final Supplier<ObjectNode> configSource;
```

Comment:
If we're going to make this private, then we should provide a protected getter so that subclasses can get their hands on the configuration.

Reply:
Is it really likely that users will subclass the FactoryPark? 
I thought the only LuthierBinderFactory is going to be extended.